data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+
+**What is a kernel?**
+In the RSP Notebook Aspect, your notebooks will be operating in a kernel that has access to the full Rubin Science Pipelines, including the "Butler" (see below) that will be your main access tool to extract images and catalogs from the DP0.1 data.
+Many standard Python libraries and modules will be available, and users can install additional Python tools they wish to use.
+In DP0.1, the Notebook Aspect will not offer access to queries from the :ref:`Tools-RSP-Portal`. 
+
+**Is all the code in Python?**
+Yes, the RSP Notebook Aspect will only have python environments for DP0.
+To access DP0.1 data from the Notebook Aspect, users will need to use Python commands and code.
+Much of the LSST Science Pipelines code is in Python, and the DP0 :ref:`Examples-DP0-1-Notebooks` use Python as well.
+These tutorials contain executable examples of the commands required to access and analyze data.
+All DP0 delegates should feel free to copy and paste from the provided tutorials.
+Anyone new to Python and looking to learn more might benefit from this `Python for Beginners <https://www.python.org/about/gettingstarted>`_ website (which includes links to tutorial in a variety of languages).
+Web searches for "python *(thing you want to do)*" are usually pretty successful too. 
+
+**What is the Butler?**
+The Butler is a middleware component of the Data Management System (DMS) for persisting and retrieving image datasets.
+The only way to access DP0.1 images is via the Butler from a Jupyter Notebook.
+The third generation "Gen3" Butler is the version being used for DP0.1.
+Full `Butler documentation <https://pipelines.lsst.io/modules/lsst.daf.butler/index.html>`_ is available, and one of the :ref:`Examples-DP0-1-Notebooks` focuses on Butler use as well. 
+
+**Where to get support with using DP0 Notebooks.**
+Keep in mind that if you are not experienced at accessing data via Jupyter notebooks, or using a Science Platform more generally, you are not alone!
+Most of the DP0 delegates are new to this environment, and all of your questions and feedback will help us improve both the documentation and the tools.
+The `DP0 Delegate Homepage` provides information about the biweekly :ref:`Delegate-Homepage-DP0-Delegate-Assemblies` which will feature live tutorials and question-and-answer time, and about :ref:`Delegate-Homepage-Getting-Support` at any time via the `Rubin Observatory Community forum <https://community.lsst.org/>`_. 
+
+
+.. _NB-Intro-Other_Docs:
+
+Additional RSP Notebook Documentation
+=====================================
+
+Additional documentation for the RSP Notebook Aspect deployed at the National Center for Supercomputing Applications (NCSA) is available at `nb.lsst.io <https://nb.lsst.io/>`_, but beware that much of it does not apply to the RSP Notebook Aspect deployed in the Google Cloud for DP0.
```

Comment:
This is actually not true anymore.  nb.lsst.io is specific to the IDF.  There are other versions for the other deployments, but the default site is for data.lsst.cloud